lms/djangoapps/course_api/blocks/api.py

```diff
@@ -17,7 +17,9 @@ def get_blocks(
         requested_fields=None,
         block_counts=None,
         student_view_data=None,
-        return_type='dict'
+        return_type='dict',
+        block_types=None,
```

Comment:
@nasthagiri When the api is called with block_counts filtered by certain block types, did you still want the rest of the blocks without the counts?

It looks like this is introducing a block type filter for all of the results, and could be in conflict with the block_counts block types.  What is best?:
1. Introduce this new overall filter and ignore/error on block types included in block_counts, but not included in block_types?
2. Always include all block types, and add block_lti_urls to match block_counts for how to turn on/off fields for specific block types?
3. Introduce new over filter and change block_counts to block_count True/False for including.  I'm guessing it's too late for this, and I'm not sure if it's what is wanted.

Thoughts?